backend/de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -1623,7 +1623,7 @@ public String DropShipPartner(final ICalloutField calloutField)
 		}
 		else
 		{
-			if (!order.isDropShip())
+			if (order.isDropShip())
```

Comment:
that `!` is here for a long time. I have problems grasping what the code actually does.
Please elaborate how removing the `!` fixes the current issue without adding other problems

Reply:
LE: please add those explanations as java-comments

Reply:
LE II but i still don't get it. if the purchase order is a dropship-order then why should
the drops-ship-location be set to the current org's warehouse after all?